### ISSUE TITLE
Missing style

### ISSUE THREAD

**Issue by martinszeltins** [icehaunter/vue3-datepicker#5]
I'm using: Vue 3 + Vite and everything works except that there is not style at all.

**Screenshot**
![Style](https://i.imgur.com/FbzYKPE.png)

```html
<template>
    <datepicker
        v-model="selected"
    />
</template>

<script>
    import { ref } from 'vue'
    import Datepicker from 'vue3-datepicker'

    export default {
        components: {
            Datepicker,
        },

        setup() {
            const selected = ref('')
            return { selected }
        }
    }
</script>
```

**Comment by RetaxMaster** [comment]
Can you solve it? I have the same issue

**Comment by Dygerydoo** [comment]
Just do:

`import 'vue3-datepicker/dist/vue3-datepicker.css';`

**Comment by icehaunter** [comment]
What @Dygerydoo specified works. I didn't want to bundle in the CSS for ease of inclusion into different build tools and ease of modification. If anyone has other opinions - I'd be glad to hear them.

**Comment by IvanCaceres** [comment]
@icehaunter you have to include the CSS just scope it tightly to the component

Otherwise people like me are importing the datepicker and it looks completely broken.

**Comment by ddavisfw** [comment]
I noticed this immediately as well. It may help to through this in the readme with a real-life sample component that includes it.

**Comment by icehaunter** [comment]
I've addressed your issue in https://github.com/icehaunter/vue3-datepicker/commit/06839f08fb3941abe682d442157daba293f6c4df by adding CSS to the JS bundle and released that as [v0.2.0](https://github.com/icehaunter/vue3-datepicker/releases/tag/v0.2.0). It shouldn't require CSS inclusion any further.

I am still unsure if that is a good idea, or if I should've just added "import this CSS" to the docs, but we'll see.
